core/bones/base.py

```diff
@@ -1192,25 +1192,36 @@ def iter_bone_value(
             else:
                 yield None, None, value
 
-    def _compute(self, skel: 'viur.core.skeleton.SkeletonInstance', name: str):
+    def _compute(self, skel: 'viur.core.skeleton.SkeletonInstance', bone_name: str):
         """Performs the evaluation of a bone configured as compute"""
 
-        if "skel" not in inspect.signature(self.compute.fn).parameters:
+        compute_fn_parameters = inspect.signature(self.compute.fn).parameters
+        compute_fn_args = {}
+        if "skel" in compute_fn_parameters:
+            cloned_skel = skel.clone()
+            cloned_skel[bone_name] = None  # remove value form accessedValues to avoid endless recursion
+            compute_fn_args["skel"] = cloned_skel
+
+        if "bone" in compute_fn_parameters:
+            parameter_found = True
+            compute_fn_args["bone"] = getattr(skel, bone_name)
+
+        if "bone_name" in compute_fn_parameters:
+            compute_fn_args["bone_name"] = bone_name
+
+        if compute_fn_args:
+            ret = self.compute.fn(**compute_fn_args)
+        else:
             # call without any arguments
             ret = self.compute.fn()
```

Comment:
Expanding an empty dictionary results in no parameter. So the `if' is superfluous.

```suggestion
        ret = self.compute.fn(**compute_fn_args)
```